### DISCUSSION TITLE
アイテムテクスチャの作成状態公表

### DISCUSSION THREAD

**Discussion by NePonpon** [TUSB/TheUnusualSkyBlock#106] [Idea]
# 注意：このディスカッションにはv13のネタバレ要素が強く含まれています。

[v13アイテム 一般公開用](https://docs.google.com/spreadsheets/d/1fBxkWnq9V46tbjBDnvVCNa7tp2IWUbw4t3IZ5pvMfdo/edit?usp=sharing)のページには**v13で実装予定のアイテムがすべて羅列**されています。  
`Texture`列には作成されたテクスチャの画像が表示されます。こちらに表示されていないアイテムに関してはテクスチャが作成されてないアイテムとなります。

## テクスチャの作成
`cmd予定`となっている数値を使用してテクスチャを作成してください。

## 作成したテクスチャについて
作成したテクスチャはサークルTUSBへ提出する必要がありません。

**Comment by anminmakura** [comment]
リソースパック製作においてご意見等があればここでコメントをして頂いても問題ないですー

**Reply by Masu-Masu** [reply]
CMD予定14のRainbowを作ってみました
使い方など全てお任せします
https://github.com/Masu-Masu/Rainbow-
